website/docs/d/secure_drift_policy.md

```diff
@@ -46,7 +46,7 @@ In addition to all arguments above, the following attributes are exported:
 
 ### Actions block
 
-The actions block is optional and supports:
+The actions block is optional and supports the following for agent versions 12.20 and above:
```

Comment:
Nit: Prevent Drift is supported for older agents as well, only container actions and capture have been added after 12.20. Same for malware